image/util.go

```diff
@@ -42,6 +42,16 @@ func GetImageLayerDirs(image *v1.Image) (res []string, err error) {
 	return
 }
 
+//get docker image layer hash path
+func GetBaseDockerImageLayerDir(image *v1.Image) (res []string) {
```

Comment:
I guess base is the "diff" at build stage? especially registry cache layer. So I think "base" will make people feel confused. Docker has Base concept, but this is not "docker base", maybe "diff" "fsdiff" "build_diff" or something would be better.